frontend-mobile.md

```diff
@@ -2,43 +2,42 @@
 
 ## About the Challenge 
 
-We're using GraphQL here, so that's gonna be the way you're going to receive the data (sounds good, right?).
+We're using GraphQL here (give a look to https://www.graphql.com/guides/), so that's gonna be the way you're going to receive the data (sounds good, right?).
```

Comment:
```suggestion
We're using GraphQL here (take a look at https://www.graphql.com/guides/), so that's gonna be the way you're going to receive the data (sounds good, right?).
```